modular_skyrat/modules/modular_implants/code/soulcatcher/soulcatcher_items.dm

```diff
@@ -0,0 +1,143 @@
+/obj/item/soulcatcher_item
```

Comment:
I still really feel like this item goes a little beyond the scope of the soulcatcher NIF software I was told about. This will have a far bigger impact on the round, and much bigger balance implications, and I'll think some more over it, but I'm starting to think it might be better off being in a separate PR so that it can be properly evaluated.

Reply:
While I can understand your concerns, I'd argue that this is something crucial for soulcatchers to work from a medical point of view. Without this item being present those without a NIF are unable to use soulcatchers, additionally, without this item, bodies are unabled to be scanned, keeping the identity of the souls that died within a round hidden.